dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/teradata/TeradataLogsConnector.java

```diff
@@ -215,15 +224,21 @@ private String getSql(@Nonnull JdbcHandle handle) {
                 // buf.append(" LEFT OUTER JOIN ").append(queryTable).append(" ST ON L.ProcID=ST.ProcID AND L.CollectTimeStamp=ST.CollectTimeStamp AND L.QueryID=ST.QueryID");
             }
 
-            buf.append(String.format(" WHERE L.UserName <> 'DBC'\n" // not useful (according to a customer)
-                    + "AND L.ErrorCode=0\n" // errors are not useful to us
+            buf.append(String.format(" WHERE L.ErrorCode=0\n"
                     + "AND L.CollectTimeStamp >= CAST('%s' AS TIMESTAMP)\n"
                     + "AND L.CollectTimeStamp < CAST('%s' AS TIMESTAMP)\n",
                     SQL_FORMAT.format(interval.getStart()), SQL_FORMAT.format(interval.getEndExclusive())));
 
-            if (!Strings.isNullOrEmpty(condition))
-                buf.append(" AND ").append(condition);
+            if (!conditions.isEmpty()) {
```

Comment:
This `if` is redundant and can be removed.

Reply:
done